src/tests/Microsoft.PowerFx.Core.Tests/DisplayNameTests.cs

```diff
@@ -763,5 +763,24 @@ public void AggregateFieldTypeDisplayName()
             var recordType = RecordType.Empty().Add("numLogical", FormulaType.Number, "numDisplay");
             Assert.Equal("numDisplay", recordType.GetFieldTypes().First().DisplayName);
         }
+
+        [Theory]
+        [InlineData("=var1", true)]
+        [InlineData("Var1", false)]
+        [InlineData("text ${var1} text", true)]
+        [InlineData("={'prop1':var1}", true)]
```

Comment:
>={'prop1':var1} [](http://example.com/codeflow?start=21&length=15)

can we assert the final string for an exact match? 

(ie, ensure things like we've preserver the TextFirst status and didn't convert to non-text first)